### PR TITLE
Update resources, add colors section

### PR DESCRIPTION
**Pull request by kartik918**
Added more relevant and new resources
Added a new colors section
Removed one inaccessible link

### CODE REVIEW

File: resources.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ Some of these platforms also take submissions, so if you designed a font with an
 * [New York Public Library Public Domain
 Images](http://www.nypl.org/research/collections/digital-collections/public-domain?hspace=331354)
 * [unDraw](https://undraw.co/)
+* [Interfacer](https://interfacer.xyz/)
```
Comment (jancborchardt) [inline resources.md:58]: Just looked through a bunch of resources from interfacer and most of them didn’t have clear licenses on where to use it. We should try and focus on not a _mass_ of resources, but about the best ones which have clearly stated open licenses like Creative Commons (CC0, Attribution, Share-Alike), or MIT, BSD, AGPL and the like.

**Comment by jancborchardt** [comment]
I’m a bit hesitant to just add a whole bunch of resources which have very little to do with open source and are just plain design resources. We should specifically focus on design at the intersection of open source, and if we just add lots and lots of general design resources we devalue the other ones who are more focused.

Anyone can easily look for general design resources, but we should focus.

**Comment by kartik918** [comment]
I agree with your point on having open licensed resources only @jancborchardt. 

I'll update and review each resource again distinctly in a follow up PR soon!

Also, what do you think about the colors section? Since that isn't technically a licensed resource.

**Comment by jancborchardt** [comment]
> Also, what do you think about the colors section? Since that isn't technically a licensed resource.

Similar to "Inspiration", I’d say let’s cut it to focus on the other things. Material Design, Bootstrap or PatternFly for example are very specifically open source and also have color systems.

**Comment by kartik918** [comment]
So I should merge "colors" and "inspiration" I presume?

Correct me if I have the wrong idea of what you have in mind @jancborchardt

**Comment by jancborchardt** [comment]
@kartik918 I’d say we remove both "colors" and "inspiration", as none of the entries are really related to open source. What do you think?

**Comment by kartik918** [comment]
@jancborchardt I am with your opinion on removing <b>inspiration</b> section since the lines are blurry on whether it could be counted as "Open Source" or not. But the section already existed before my PR so I'm not sure if I should remove something that existed before my addition completely.

About the <b>colors</b>, I think it should be there because colors aren't "claimed" or copyrighted in general. Providing open source color contrast tools to help out in designs seems like a good choice. 

I'll refine my list to include only open source color tools but if you still want it removed, let me know.

**Comment by jancborchardt** [comment]
> But the section already existed before my PR so I'm not sure if I should remove something that existed before my addition completely.

That’s fine, as we can always improve on existing stuff. :) That also means deletion, not only addition.

> Providing open source color contrast tools to help out in designs seems like a good choice.

That would indeed be good! So it’s best to check which of the tools are open source and mainly keep those, right?

Thanks for your contributions! :)

**Comment by kartik918** [comment]
@jancborchardt thanks for clearing all that up, I'll open a PR with the appropriate OS color tools soon!

P.S could you send an invite to me for joining the org here, I can then contribute directly in the future (and represent 🏳‍🌈). That would be super, thanks!

**Pull request merged**